docs/hosting-a-server-with-northstar/prerequisites.md

```diff
@@ -1,6 +1,8 @@
 # Prerequisites
 
-**TL;DR:** Port forward `37015` (UDP). If using Northstar v1.12 or older, also port forward `8081` (TCP).
+**TL;DR:** Port forward `37015` (UDP), and no [CGNAT](prerequisites.md#cgnat)
+
+Note that by default your server will be called ``Unnamed Northstar Server``. You can change the name as described in the next page.
```

Comment:
I'd drop the second line byline. Keep it short and simple. The TL;DR is kinda intended for someone that hosted servers for other games before. So even CGNAT mention might be overkill

```suggestion
**TL;DR:** Port forward `37015` (UDP), and no [CGNAT](prerequisites.md#cgnat)
```